R/Viz.R

```diff
@@ -34,7 +34,7 @@ PlotAvgExpr <- function(GenesNames2Show, X_avg, Y_avg, features=NULL, Xlab="Xlab
   p1 <- ggplot(avg.combo.cells, aes(X, Y)) + geom_point() + 
     geom_text(aes(label=gene3), size=3, colour=HighColor,
               vjust=0, hjust=-0.1) +
-    ggtitle(Title) + xlab(Xlab) + ylab(Ylab) + 
```

Comment:
Why do these need to change?  Isnt this a basic ggplot, not patchwork?  ggtitle() itself is perfectly fine to use.  the problem is that FeaturePlot and some similar functions from Seurat not return patchwork plot object when there is a multi-plot (such as a feature plot with 2 features).  ggtitle no longer works direct on this.  all these in Viz.R look like simple ggplot objects?